lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -88,6 +88,7 @@ type VicContainerProxy interface {
 
 	Stop(vc *viccontainer.VicContainer, name string, seconds *int, unbound bool) error
 	IsRunning(vc *viccontainer.VicContainer) (bool, error)
+	IsBroken(vc *viccontainer.VicContainer) (bool, error)
```

Comment:
Are we sure this will only be needed in the container endpoints? I am wondering if we will need this information for networks or volumes at some point.